tests/WebSharper.Tests/DateTime.fs

```diff
@@ -231,6 +231,294 @@ let Tests =
             isTrue (DateTime.IsLeapYear 2020)
             isFalse (DateTime.IsLeapYear 2100)
         }
+
+        // ToString with custom format test is split in different parts
+        // otherwise the compilation hangs for ever (seems to be a bug)
+
+        Test "ToString with custom format works (Part #1)" {
+            equal (DateTime(2014, 7, 1, 16, 37, 1, 2).ToString("r d")) "r 1"
+            equal (DateTime(2014, 7, 13, 16, 37, 1, 2).ToString("r d")) "r 13"
+
+            equal (DateTime(2014, 7, 1, 16, 37, 0).ToString("r dd")) "r 01"
```

Comment:
Like mentioned in this comment I was not able to put all the tests under a single `Test "..."` call otherwise the compilation  hanged for ever.